staff/felipe-fernandez/precourse/calculator2/index.js

```diff
@@ -0,0 +1,480 @@
+    datazerobig.appendChild(node2);
+    numberTotal += '2';
+}
+
+threeButton.addEventListener("click", functionNumber3);
+
+function functionNumber3() {
+   
+    var number3=3;
+    var node3 = document.createTextNode(number3);
+    datazerobig.appendChild(node3);
+    numberTotal += '3';
+}
+
+fourButton.addEventListener("click", functionNumber4);
+
+function functionNumber4() {
+   
+    var number4=4;
+    var node4 = document.createTextNode(number4);
+    datazerobig.appendChild(node4);
+    numberTotal += '4';
+}
+
+fiveButton.addEventListener("click", functionNumber5);
+
+function functionNumber5() {
+   
+    var number5=5;
+    var node5 = document.createTextNode(number5);
+    datazerobig.appendChild(node5);
+    numberTotal += '5';
+}
+
+sixButton.addEventListener("click", functionNumber6);
+function functionNumber6() {
+   
+    var number6=6;
+    var node6 = document.createTextNode(number6);
+    datazerobig.appendChild(node6);
+    numberTotal += '6';
+}
+
+sevenButton.addEventListener("click", functionNumber7);
+function functionNumber7() {
+   
+    var number7=7;
+    var node7 = document.createTextNode(number7);
+    datazerobig.appendChild(node7);
+    numberTotal += '7';
+}
+
+eightButton.addEventListener("click", functionNumber8);
+function functionNumber8() {
+   
+    var number8=8;
+    var node8 = document.createTextNode(number8);
+    datazerobig.appendChild(node8);
+    numberTotal += '8';
+}
+
+nineButton.addEventListener("click", functionNumber9);
+function functionNumber9() {
+   
+    var number9=9;
+    var node9 = document.createTextNode(number9);
+    datazerobig.appendChild(node9);
+    numberTotal += '9';
+}
+
+numberzeroButton.addEventListener("click", functionNumber0);
+function functionNumber0() {
+   
+    var number0=0;
+    var node0 = document.createTextNode(number0);
+    datazerobig.appendChild(node0);
+    numberTotal += '0';
+}
+
+//Borrar el marcador
+ac.addEventListener("click", functionAc);
+function functionAc() {
+
+    divideDisplay=0;
+    multiplyDisplay=0;
+    sumDisplay=0;
+    restDisplay=0;
+    datazerobig.innerHTML='';
+    numberTotal = '';
+    total=0;
+    numberComas=0;
+    numberPoints=0;
+}
+
+//Quitar el último caracter del marcador
+
+undo.addEventListener("click", functionUndo);
+function functionUndo() {
+   
+    numberTotal=numberTotal.slice(0, -1);
+    var actualNode = document.createTextNode(numberTotal);
+    datazerobig.innerHTML='';
+    datazerobig.appendChild(actualNode);
+    //Cuando se borre un número con coma que se vuelva a poder poner coma
+    if (!numberTotal.includes('.')){
+        numberComas=0;
+        numberPoints=0;
+    }
+}
+
+//función coma
+var numberComas=0;
+var numberPoints=0;
+
+coma.addEventListener("click", functionComa);
+function functionComa() {
+   
+    var coma=',';
+    lastCharacterComa=numberTotal.slice(-1);
+    //que no se pueda añadir más de una coma (punto) seguidas
+    if (lastCharacterComa=== '.'){
```

Comment:
Why this if statement? Itś empty.

Reply:
Ifs empty changed with a return null statement.